I.write-tests/05-get-second-third-smallest/get-second-third.test.js

```diff
@@ -1,5 +1,15 @@
 // example
 // input = [90, 5, 11, 8, 6];
 // expected output = [6, 8];
+// also test that the original array has not changed
 
-// also test that the original array has not changed
\ No newline at end of file
+var secondThird = require("./get-second-third");
+test("Get second and third numbers", function() {
+  // Given
+  var nums = [90, 5, 11, 8, 6];
+  // When
+  var result = secondThird(nums);
+  // Then
+  expect(result).toEqual([6, 8]);
+  expect(nums).toEqual(nums); // MENTORS - Advise is this is the way to test and see if new array has not changed.
```

Comment:
Not quite right here. This will always pass since the variable will always be equal to itself, even though it may have changed over time. What you really want to test is that the original array still has all the original values. 

Generally, tests should compare a variable to something whose value is completely specified. For example, in Line 13, you are correctly comparing variable "result" to a **very specific value "[6, 8]"**. Similarly, what can you compare variable "nums" to ?

Reply:
I could compare it the given array, but what if the array changes. the test may fail because I kind of hard coded the array in the test.

Reply:
Great question. I would say, in the case of tests like this, it's ok to "hard code" expected values. After all, you want to test your code against test cases that are as specific and explicit as possible. This reduces chances of making a mistake in the test code itself.

However, you make a good point about not wanting to repeat the array. In order to make that work, you need a way to preserve the original numbers array (call it `originalNums`) and pass a ***separate*** array (call it `inputNums`) to the secondThird function. That way, you can compare `originalNums` to `inputNums` using expect/toEquals.